gr-fft/include/gnuradio/fft/window.h

```diff
@@ -330,8 +330,10 @@ class FFT_API window
      * \param type a gr::fft::win_type index for the type of window.
      * \param ntaps Number of coefficients in the window.
      * \param beta Used only for building Kaiser windows.
+     * \param normalize If true, return a window with unit power
      */
-    static std::vector<float> build(win_type type, int ntaps, double beta);
+    static std::vector<float>
+    build(win_type type, int ntaps, double beta = 6.76, const bool normalize = false);
```

Comment:
Does this become awkward for a window that doesn't require a beta value, but user wants it normalized?  Should there be an overloaded build that has normalize as the third parameter.

Reply:
That's a good suggestion. But note that in Python, you're already golden because of the combo of PyBind11 and kwargs. This is legit:
```python
win = fft.window.build(fft.win_type.WIN_BLACKMAN_hARRIS, 21, normalize=True)
```

Reply:
I would say, you're maybe right,  but we can add a second `build()` function if someone wants it in a different PR.